### PR TITLE
docs: fix compile errors; rename variables to match impl

### PR DESCRIPTION
**Pull request by activedecay**
- exported the controller class
- the `schema:` prop was missing
- the variable names were mismatched with types declared and 
  exposed through openapi/swagger/explorer thingie

### CODE REVIEW

File: docs/site/Decorators_openapi.md [hunk 1]
```diff
@@ -373,14 +380,47 @@ const schemaWithArray = {
...
+      },
+    },
+  })
+  everyone(): MyModel[] {
+    return [{name: 'blue'}, {name: 'red'}];
+  }
+
+  @post('/my-controllers')
+  greetEveryone(
+      @requestBody({
```
Comment (raymondfeng) [inline docs/site/Decorators_openapi.md]: Two spaces.

File: docs/site/Decorators_openapi.md [hunk 2]
```diff
@@ -373,14 +380,47 @@ const schemaWithArray = {
...
+        content: {'application/json': {schema: schemaDeepArrayOfMyModel}},
+      })
+          body: MyModel[][],
```
Comment (raymondfeng) [inline docs/site/Decorators_openapi.md]: Misaligned.

**Comment by raymondfeng** [comment]
@activedecay Would you please rebase on https://github.com/strongloop/loopback-next/pull/1937? We probably should land #1937 1st.

**Comment by activedecay** [comment]
oh! ofc. thanks

**Comment by raymondfeng** [comment]
Please run `npm lint:fix` to fix the style/formatting issue - https://travis-ci.org/strongloop/loopback-next/jobs/448008086

**Comment by activedecay** [comment]
:100: cool. 

now that i've broken travis/lint i know how to fix it. thank you

**Comment by raymondfeng** [comment]
Landed as https://github.com/strongloop/loopback-next/commit/14d9419b835f6b34b80f08cec6240496f25719a5 after squashing the commits.

**Pull request closed without merging**